membership_group/models/membership_group_member.py

```diff
@@ -18,11 +24,55 @@ class MembershipGroupMember(models.Model):
             ("committee", "Committee"),
         ],
     )
+    date_from = fields.Date(
```

Comment:
Can we make the strings of these fields like this:
date_from: `string="From"`
date_to: = `string="To"`
date_end: =`string="Ended on"`

I often think "Date" is unnecesarily redundant and taking up screen real estate